doc/devel/contributing.rst

```diff
@@ -36,17 +89,16 @@ Contributor incubator
 ---------------------
 
 The incubator is our non-public communication channel for new contributors. It
-is a private gitter room moderated by core Matplotlib developers where you can
-get guidance and support for your first few PRs. It's a place you can ask
-questions about anything: how to use git, GitHub, how our PR review process
-works, technical questions about the code, what makes for good documentation
-or a blog post, how to get involved in community work, or get
+is a private gitter_ (chat) room moderated by core Matplotlib developers where
+you can get guidance and support for your first few PRs. It's a place where you
+can ask questions about anything: how to use git, GitHub, how our PR review
+process works, technical questions about the code, what makes for good
+documentation or a blog post, how to get involved in community work, or get a
```

Comment:
blog post?

Reply:
Yes there is a blog 😄 https://blog.scientific-python.org/tags/matplotlib/